tools/src/podio-dump-tool.cpp

```diff
@@ -70,19 +70,21 @@ std::vector<size_t> parseEventRange(const std::string_view evtRange) {
   };
 
   // Split by ',' and transform into a range of string views
-  auto splitRange = evtRange | rv::split(',') |
-      rv::transform([](auto&& subrange) { return std::string_view(subrange.begin(), subrange.end()); });
+  auto splitRange = evtRange | rv::split(',') | rv::transform([](auto&& subrange) {
+                      return std::string_view(&*subrange.begin(), std::ranges::distance(subrange));
```

Comment:
This is a bit ugly, maybe it would be good to have a preprocessor check with the current and new version to make cleanup easier and then no one has to wonder why it's done like this. Same for below.

Reply:
I moved it into a `splitString` utility function where there is now a pre-processor differentiation.